package.json

```diff
@@ -26,7 +26,7 @@
     "http-proxy": "^1.18.1",
     "lodash": "^4.17.15",
     "lodash-uuid": "^0.0.3",
-    "material-table": "1.69.1",
+    "material-table": "2.0.6",
```

Comment:
@delcroip this won't work. We need to stay on 1.69.1 until material-ui is upgrade to v5 see: https://github.com/openimis/openimis-fe_js/pull/183 u